docs/source/quantization-backend-configuration.rst

```diff
@@ -0,0 +1,26 @@
+Quantization Backend Configuration
+----------------------------------
+
+FX Graph Mode Quantization allows the user to configure various
+quantization behaviors of an op in order to match the expectation
+of their backend.
+
+In the future, this document will contain a detailed spec of
+these configurations.
+
+
+Default values for native configurations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below is the output of the configuration for quantization of ops
+in fbgemm and qnnpack (PyTorch's default quantized backends).
+
+Code to generate results::
+
+  from torch.ao.quantization.fx.backend_config import get_native_backend_config_dict
+  from pprint import pprint
+  pprint(get_native_backend_config_dict())
```

Comment:
I think this would look better:
```
import json
print(json.dumps(get_native_backend_config_dict(), indent=4))
```

Reply:
Something like this would work:

```
from torch.utils._pytree import tree_map
config = get_native_backend_config_dict()
# convert Python types to strings, so json module can understand them
config_str = tree_map(str, config)
f.write(json.dumps(config_str, indent=2))
```

However, the results would look like:

```
{
  "pattern": "<function leaky_relu at 0x7fd4083abb80>",
  "observation_type": "ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT",
  "dtype_configs": [
    {
      "input_dtype": "torch.quint8",
      "output_dtype": "torch.quint8"
    }
  ]
},
```

It's easier to read, but also everything is printed as strings, which may be confusing to the user since these objects are not actually strings.  I think a custom printing function may have to be written to both make it nice on the eyes and also not print incorrect types.

Reply:
I see. Maybe we can do that in the future then